index.html

```diff
@@ -0,0 +1,240 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css"
+      integrity="sha512-Fo3rlrZj/k7ujTnHg4CGR2D7kSs0v4LLanw2qksYuRlEzO+tcaEPQogQ0KaoGN26/zrn20ImR1DfuLWnOo7aBA=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="./styles.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <main class="first-card">
+      <section class="first-card-section">
+        <div class="all-components">
+          <div class="first-image">
+            <img src="./asset/Snapshoot.png" alt="" />
+          </div>
+          <div class="tonic-content">
+            <h1>TONIC</h1>
+            <div class="tonic-content-canopy">
+              <p>CANOPY</p>
+              <ul class="tonic-list">
+                <li>Back End Dev</li>
+                <li>2021</li>
+              </ul>
+            </div>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <div class="tonic-courses">
+              <ul>
+                <li><a href="#">html</a></li>
+                <li><a href="#">css</a></li>
+                <li><a href="#">javascript</a></li>
+              </ul>
+            </div>
+            <div class="btn">
+              <button type="submit" class="btn-tonic">See Project</button>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="second-card-section">
+        <div class="all-components">
+          <div class="first-image">
+            <img src="./asset/Snapshoot(1).png" alt="" />
+          </div>
+          <div class="tonic-content">
+            <h1>Multi-post <br />Stories</h1>
+            <div class="tonic-content-canopy">
+              <p>CANOPY</p>
+              <ul class="tonic-list">
+                <li>Back End Dev</li>
+                <li>2021</li>
+              </ul>
+            </div>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
```

Comment:
- [ ] One of the requirements says that you should use HTML semantic tags when structuring your project. Kindly use a better structure by using `<ul>` to list all of your work projects and wrap your `<ul>` in one section.
Your sturcture
![image](https://user-images.githubusercontent.com/65068771/149473817-9d45a262-65be-44f0-9a61-aa70541f9a9d.png)

A better structure example
```html
<header>
    ....content
</header>
<main>
   <section class="home-section">.... content</section>
   <section class="work-section">
      <ul class="cards"> <-- This should use grid to layout its content -->
         <li class="card">....content</li>
         <li class="card">....content</li>
         <li class="card">...content</li>
         <li class="card">....content</li>
      <ul>
   </section>
   <section class="about-section">...content</section>
   <section class="contact section">...content</section>
</main>

```